gm_dotnet_managed/GmodNET/GameConsoleWriter.cs

```diff
@@ -13,9 +14,53 @@ internal class GameConsoleWriter : TextWriter
         public override string NewLine => "\n";
         public override Encoding Encoding => Encoding.UTF8;
 
-#pragma warning disable CA2101
+        public GameConsoleWriter(ILua lua)
+        {
+            if (OperatingSystem.IsWindows())
+            {
+                Msg = Msg_Tier0;
+            }
+            else if (OperatingSystem.IsLinux())
+            {
+                lua.PushSpecial(SPECIAL_TABLES.SPECIAL_GLOB);
+                lua.GetField(-1, "game");
+                lua.GetField(-1, "IsDedicated");
+                lua.MCall(0, 1);
+                bool is_dedicated = lua.GetBool(-1);
+                lua.Pop(1);
+
+                if (is_dedicated is true)
```

Comment:
This new C# 9.0 syntax is fancy, but I would rather just write `if (is_dedicated)`.

Reply:
~~every day we stay futher from god~~
yeah, i'm just using `is` and `is not` just too often.